app/views/products/_comments.html.erb

```diff
@@ -10,7 +10,7 @@
       <% end %>
       <% unless current_user.nil? %>
         <% if signed_in? && (current_user.admin? || current_user.moderator?) %>
-          <%=  render partial:'comment' %>
+          <%=  render partial:"comments/comment", locals: { comment: comment } %>
```

Comment:
Well... No idea why it didn't work without locals on Heroku :s !

But it's working ^^.

I will take a look later and I will try to figure out :)

![Programmers dilema of life](https://s-media-cache-ak0.pinimg.com/236x/6c/05/cf/6c05cfeab116063ce79a0dd4899debff.jpg)